doc/authoring_command_modules/README.md

```diff
@@ -75,63 +75,77 @@ Command modules should have the following structure:
 |           `-- <MODULE_NAME>
 |               `-- __init__.py
 `-- setup.py
+`-- HISTORY.rst
 ```
 
 **Create an \_\_init__.py for your module**
 
-In the \_\_init__ file, two methods need to be defined:
-  - `load_commands` - Uses the file in the 'Writing a Command' section below to load the commands.
-  - `load_params` - Uses the file in the 'Customizing Arguments' section below to load parameter customizations.
+In the \_\_init__ file, you will declare a command loader class that inherits from AzCommandsLoader. You will typically override the following three methods:
+  - `__init__` - Useful for setting metadata that applies to the entire module. See command authoring for more info.
```

Comment:
Would it be worthwhile to mention that no heavy processing is expected here (for performance reasons)?

Reply:
Sure, I can add that, but I think ultimately we'll need an automated test that detects these performance regressions and blocks PRs that introduce them.